cost/azure/unused_sql_databases/azure_unused_sql_databases.pt

```diff
@@ -200,14 +200,19 @@ script "js_merged_metrics", type: "javascript" do
             total += ts_data[x].count;
           }
           if(total == 0){
+            if(databases_metrics[i].sku === null || typeof databases_metrics[i].sku === "undefined"){
+              var sku={};		
```

Comment:
instead of leaving it blank can we use Unknown or Not Available?

Reply:
I have added "Not available" as value.